apps/next/components/user/login-form.tsx

```diff
@@ -33,7 +34,31 @@ import { Switch } from "@/components/ui/switch"
 
 import { signInWithEmailPassword, signInWithOtp } from "@/modules/user/auth"
 
-export const LoginForm: React.FC = () => {
+const getErrorProps = (
+  passwordlessSignInError: Error | null,
+  signInError: Error | null,
+  error: SearchParamError | null
+) => {
```

Comment:
I think this function should only take an Error or a SearchParamsError. For clarity, we should make the argument an object: { error, searchParamsError } and use the OR operator when calling the function.